.github/workflows/update_robottelo_image.yml

```diff
@@ -0,0 +1,97 @@
+name: update_robottelo_image
+
+on: [push]
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
+    env:
+      PYCURL_SSL_LIBRARY: gnutls
+    steps:
+      - name: Checkout Robottelo
+        uses: actions/checkout@v2
+
+      - name: Set Up Python-${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install -y libgnutls28-dev libcurl4-openssl-dev libssl-dev
+          wget https://raw.githubusercontent.com/SatelliteQE/broker/master/broker_settings.yaml.example
+          pip install -U --no-binary=pycurl -r requirements.txt -r requirements-optional.txt
+          cp robottelo.properties.sample robottelo.properties
+          cp broker_settings.yaml.example broker_settings.yaml
+          cp virtwho.properties.sample virtwho.properties
+
+      - name: Pre Commit Checks
+        uses: pre-commit/action@v2.0.0
+
+      - name: Collect Tests
+        run: |
+          pytest -n 8 --setup-plan --disable-pytest-warnings tests/foreman/ tests/robottelo/
+          pytest -n 8 --setup-plan --disable-pytest-warnings -m pre_upgrade tests/upgrades/
+          pytest -n 8 --setup-plan --disable-pytest-warnings -m post_upgrade tests/upgrades/
```

Comment:
might be useful? or required? to `export SATELLITE_VERSION=6.9` to avoid ssh checks for version as there is no SAT instance involved. 

robottelo tests are decorated by BZs - the decorator needs to know SAT version for its own logic

Reply:
I'd actually like to fix this behavior in robottelo, so that it will default to some version automatically without needing to set this env var. If there's no host to connect to, just pick a version. Downside being we would have to periodically update it (maybe once a year)

Reply:
Mike, so I shouldn't add it here, right?

Reply:
its fine to have it here for now

Reply:
Added the SATELLITE_VERSION env variable [here](https://github.com/SatelliteQE/robottelo/pull/8150/files#diff-ee633393f1eca923b9e701f896601dcb1a7a1cd0c55238b8c0bb37c076459898R14).